.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This config won't help us do any debugging. There will be some config you can use from the other webapp.